bower.json

```diff
@@ -9,31 +9,31 @@
     "tests"
   ],
   "dependencies": {
-    "jquery": "2.1.1",
-    "underscore": "1.8.3",
-    "backbone": "1.2.3",
-    "backbone-associations": "0.6.2",
-    "bootstrap": "3.3.5",
-    "font-awesome": "4.4.0",
-    "requirejs": "2.1.20",
-    "requirejs-text": "2.0.14",
+    "jquery": "^3.2.1",
```

Comment:
We originally pinned the versions to a specific version because sometimes minor version changes caused deployments to fail.

I wonder if you could investigate using yarnpkg.org to handle pinning the versions and keep the `^` in here  (run `yarn` instead of `npm install`)